Tests/CommonMarkTests/StatisticsVisitor.swift

```diff
@@ -0,0 +1,161 @@
+import CommonMark
+
+public struct DocumentStatistics: Equatable {
```

Comment:
Any reason for this to be `public` instead of (implicitly) `internal`?

Reply:
Yes, fixed. 🙂